### DISCUSSION TITLE
VideoInfo from Video Player Returns 0 for everything

### DISCUSSION THREAD

**Discussion by chuntimchan** [PaulleDemon/tkVideoPlayer#41] [Q&A] [answered]
Hey, the title really says it all so here's my code.

![image](https://github.com/PaulleDemon/tkVideoPlayer/assets/79533731/77abb789-ac00-4663-8dee-ac4360b8beb4)

Output

![image](https://github.com/PaulleDemon/tkVideoPlayer/assets/79533731/c9d3dc44-79ea-49c1-bd90-981b9cfc6883)

**Comment by PaulleDemon** [comment, marked as answer]
Hey sorry for the late reply. Didn't receive a notification. Coming to your question. You are trying to get the video details before it has a chance to get the video information I'd suggest that you listen to  `<<Duration>>` event to know if the video info has been loaded, then display it.
eg:
```
videoplayer.bind("<<Duration>", lambda e: videoplayer.video_info())
```
